spec/live/article_spec.rb

```diff
@@ -34,6 +34,7 @@ def valid_attributes
 
         expect(actual.count).to be > 0
         expect(actual.last.title).to eq("What are these sections and articles doing here?")
+        expect(actual.last.votes).to eq([])
```

Comment:
I couldn't find a way to create an upvote (I _think_ that's an area that's completely overlooked in the gem), so this is the best I can do in terms of testing.